docs/surveys/salary-survey/2019.rst

```diff
@@ -0,0 +1,735 @@
+partially onsite.
+
+Of the respondents that work completely on site, 55% do so by choice,
+while for 45% it’s a requirement.
+
+Of those that work entirely remotely, 74% do so by choice, while only
+26% have no on site alternative available to them.
+
+.. figure:: images/8.png
+   :alt: Figure: Work Location
+
+   Figure: Work Location
+
+--------------
+
+.. h.me9iqb10zfgx:
+
+Section 2: Salary Information
+=============================
+
+This section contained questions that addressed the all-important salary
+figure, additional benefits, level of satisfaction and reasons for
+dissatisfaction.
+
+.. h.7208juau05as:
+
+Annual Salary
+-------------
+
+Note: as 97% (632) of respondents reporting working between 30 and 60
+hours per week - a “full time” role - the 3% reporting fewer than 30
+hours have been omitted from the figures in this section.
+
+The median salary across all regions was $74,500 (meaning half of the
+respondents earned more, and half earned less).
+
+This figure is perhaps not very representative as it takes into account
+the socio-economic situation of the countries of the very highest
+earners (out of the top 10 salary values, 9 were from the US and one
+from the UK) and the very lowest (the bottom 10 salary values were from
+ India, Asia, and Eastern Europe).
+
+Figures grouped into regions make a more useful baseline from which to
+determine what constitutes a “fair” salary.
+
+.. h.mqgdllu8gaap:
+
+Salary by Region/Country
+------------------------
+
+.. table::  Median Salary by Region/Country
+
+   +-----------------+-----------------+-----------------+-----------------+
+   | Region          | No of           | Percentage of   | Median Salary   |
+   |                 | Respondents     | Total           |                 |
+   |                 | (working 30+    |                 |                 |
+   |                 | hours)          |                 |                 |
+   +=================+=================+=================+=================+
+   | North America   | 386             | 61%             | $90,000         |
+   +-----------------+-----------------+-----------------+-----------------+
+   | - USA           | 344             | 54.4%           | $93,600         |
+   +-----------------+-----------------+-----------------+-----------------+
+   | - Canada        | 42              | 6.6%            | $60,273         |
+   +-----------------+-----------------+-----------------+-----------------+
+   | Europe          | 164             | 25.9%           | $48,700         |
+   +-----------------+-----------------+-----------------+-----------------+
+   | - EU Member     | 112             | 17.2%           | $46,104         |
+   | States          |                 |                 |                 |
+   +-----------------+-----------------+-----------------+-----------------+
+   | - UK \*         | 36              | 5.5%            | $60,269         |
```

Comment:
Might be nice to rejig these as proper footnotes:

```
diff --git a/docs/surveys/salary-survey/2019.rst b/docs/surveys/salary-survey/2019.rst
index 3cf3b711..340fbe98 100644
--- a/docs/surveys/salary-survey/2019.rst
+++ b/docs/surveys/salary-survey/2019.rst
@@ -319,7 +319,7 @@ Salary by Region/Country
    | - EU Member     | 112             | 17.2%           | $46,104         |
    | States          |                 |                 |                 |
    +-----------------+-----------------+-----------------+-----------------+
-   | - UK \*         | 36              | 5.5%            | $60,269         |
+   | - UK [#f1]_     | 36              | 5.5%            | $60,269         |
    +-----------------+-----------------+-----------------+-----------------+
    | - Germany       | 24              | 3.7%            | $59,430         |
    +-----------------+-----------------+-----------------+-----------------+
@@ -340,8 +340,11 @@ Salary by Region/Country
    | Given           |                 |                 |                 |
    +-----------------+-----------------+-----------------+-----------------+
 
-\* including Scotland - no respondents were reported from Wales or
-Northern Ireland
+
+.. rubric:: Footnotes
+
+.. [#f1] including Scotland - no respondents were reported from Wales or
+ Northern Ireland
```

Reply:
Yea, it's just an initial HTML<->RST import. We might move forward w/ publishing HTML, so I wanted to wait to polish it until we see how it looks.

Reply:
Yeah, that is why I didn't go ahead and fix it all.